app/src/main/java/com/mmm/parq/activities/DriverActivity.java

```diff
@@ -223,13 +215,16 @@ public boolean onNavigationItemSelected(MenuItem menuItem) {
                     case R.id.drawer_history:
                         mFragment = new DriverHistoryFragment();
                         break;
-                    case R.id.drawer_settings:
-                        //mFragment = new DriverSettingsFragment();
-                        logOut();
-                        break;
+                    // case R.id.drawer_settings:
+                    //    mFragment = new DriverSettingsFragment();
+                    //    break;
```

Comment:
Honestly this is here because Kenz left a stubbed thing there, I just figured this was a bit clearer. We can definitely take it out if we don't want it, I just didn't want to remove the comment entirely without a bit of clarification.